compiler/modules/bir/bir.bal

```diff
@@ -92,6 +94,19 @@ public type FunctionSignature readonly & record {|
     SemType? restParamType = ();
 |};
 
+public type Region record {|
+    Label entry;
+    Label? exit = ();
+    Label? parent = ();
+    RegionType ty;
+|};
+
+public enum RegionType {
+    Simple,
+    Loop,
+    Multiple
```

Comment:
Don't know what a Multiple region is.

Reply:
Multiple type was used to represent a region for a if-else construct.